app/tests/spec/lib/logger.js

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  var AuthErrors = require('lib/auth-errors');
+  var chai = require('chai');
+  var Logger = require('lib/logger');
+  var sinon = require('sinon');
+  var WindowMock = require('../../mocks/window');
+
+  var assert = chai.assert;
+
+  describe('lib/logger', function () {
```

Comment:
Should we also be asserting that the methods don't throw if the relevant console method has been clobbered? Given that they all contain that check, it feels like we should.